.travis.yml

```diff
@@ -1,25 +1,25 @@
 language: node_js
-
 sudo: false
-
 node_js:
 - 12
 - 14
-
 before_install:
+- openssl aes-256-cbc -K $encrypted_3d8a3eb98382_key -iv $encrypted_3d8a3eb98382_iv
```

Comment:
This is essentially a repeat of line 10-11, please remove

Reply:
Fixed, @kennburger